code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -329,8 +331,17 @@
 		L.handle_ventcrawl(src)
 		return
 
-
+/// Whether ventcrawling creatures can move in or out of this machine.
 /obj/machinery/atmospherics/proc/can_crawl_through()
+	// If it's broken, nothing can pass through
+	if(machine_stat & BROKEN)
+		return FALSE
+
+	// If it is powered on, but off, nothing can pass through
+	if(powered() && !on)
+		return FALSE
```

Comment:
This makes no sense, why would someone not be able to crawl through a scrubber or vent that is turned off?

Reply:
If people want to block vent crawlers, they should have to weld the vents

Reply:
If it is powered and turned off it is presumably closed to the point of being air tight.

Reply:
the vent has motors that (while being powered) will force air flaps to close, meaning nothing can go through them